pkg/sdk/go/channels_test.go

```diff
@@ -41,21 +45,21 @@ func TestCreateChannel(t *testing.T) {
 	}{
 		{
 			desc:    "create new channel",
-			channel: channel,
+			channel: channel1,
```

Comment:
Here you should not use a Channel with external ID. Create a new one `chExtID` and use it only in the corresponding tests.

Reply:
Done